dashboard/test/models/user_level_test.rb

```diff
@@ -215,10 +218,13 @@ class UserLevelTest < ActiveSupport::TestCase
   test 'other changes do not destroy unclaimed peer reviews' do
     level = create(:free_response, peer_reviewable: true)
     script = create :script
+    level_source = create :level_source
 
     ul = UserLevel.create(
       user: @user,
       level: level,
+      script: script,
+      level_source: level_source,
```

Comment:
A couple of test cases in this file started failing after my change - I actually suspect they were never testing what they claimed to, because they were silently hitting the now-removed edge case before running the code to actually create a peer-reviewable UserLevel.